### DISCUSSION TITLE
Unable to install phppuppeteer need help

### DISCUSSION THREAD

**Discussion by sajedulsakib001** [rialto-php/puphpeteer#168] [Q&A]
I'm trying to install phppuppeteer.. but i am getting this error can anyone help me please.. 

[13-Feb-2022 10:51:29 UTC] PHP Fatal error:  Uncaught Error: Class 'Nesk\Puphpeteer\Puppeteer' not found in /home/...../public_html/a.php:5
Stack trace:
#0 {main}
  thrown in /home/..../public_html/a.php on line 5

**Comment by tanz992006** [comment]
I'm having the same error when trying to test the script.
Please help.
